### PR TITLE
Refactor toto to in-toto - Fix #17

### LINKED ISSUE #17: Refactor framework name 'toto' --> 'in-toto'
We decided to rename the framework to 'in-toto' (lat. as a whole) to avoid collisions with existing software projects.
We need to refactor the code base accordingly.

### PR DESCRIPTION
**Pull request by lukpueh**
Finally refactors toto into **in-toto** everywhere. Fixes https://github.com/in-toto/in-toto/issues/17

- Repo name: in-toto
- Package setup.py name: in-toto (pip replaces the hyphen with an underscore)
- Package directory name: in_toto (`import`able as opposed to packages/modules with hyphen)
- CLI interfaces (`import`able)
  - in_toto_run.py
  - in_toto_record.py
  - in_toto_verify.py
with entry points (console scripts) specified in `setup.py`
  - in-toto-run
  - in-toto-record
  - in-toto-verify

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,17 +5,24 @@
...
-  install_requires=['six', 'simple-settings', 'attrs', 'canonicaljson',
-                    'python-dateutil', 'iso8601'],
+  install_requires=["six", "simple-settings", "attrs", "canonicaljson",
+                    "python-dateutil", "iso8601"],
   test_suite="test.runtests",
-  scripts=['toto/toto-run.py',
-           'toto/toto-verify.py']
+  scripts=["in_toto/in_toto_run.py",
+           "in_toto/in_toto_record.py",
+           "in_toto/in_toto_verify.py"],
```
Comment (SantiagoTorres) [inline setup.py]: do we still need this section if the entry_points are defined?

File: tox.ini [hunk 2]
```diff
@@ -17,5 +17,5 @@ deps =
 commands =
     # Integrating tox with setup.py test is as of Oct 2016 discouraged
     # http://tox.readthedocs.io/en/latest/example/basic.html#integration-with-setup-py-test-command
-    # coverage run --source=toto setup.py test
+    # coverage run --source=in-toto setup.py test
```
Comment (SantiagoTorres) [inline tox.ini]: do we want/need this line?

**Review by SantiagoTorres — CHANGES_REQUESTED** [review]
Two minor nits, but overall is what we need 👍

**Comment by JustinCappos** [comment]
Glad to see it getting addressed!

**Review by SantiagoTorres — APPROVED** [review]

**Comment by SantiagoTorres** [comment]
thanks @lukpueh!

**Pull request merged**